src/pages/Inquiry/Inquire/components/InquireForm/index.tsx

```diff
@@ -1,24 +1,100 @@
-import { useState } from 'react';
+import {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+import { useLocation } from 'react-router-dom';
 
+import { InquiryImage, SubmitInquiry } from 'api/inquiry/entity';
+import { ReactComponent as DeleteIcon } from 'assets/svg/inquiry/image-delete.svg';
 import { ReactComponent as UploadIcon } from 'assets/svg/inquiry/image-upload.svg';
 import ToggleButton from 'components/common/ToggleButton';
-import RequiredLabel from 'pages/Inquiry/Inquire/components/InquireForm/components/RequiredLabel/index';
-import useBooleanState from 'utils/hooks/useBooleanState';
+import useSubmitInquiry from 'pages/Inquiry/hooks/useSubmitInquiry';
+import RequiredLabel from 'pages/Inquiry/Inquire/components/InquireForm/RequiredLabel';
+import cn from 'utils/ts/classNames';
+import makeToast from 'utils/ts/makeToast';
 
 import styles from './InquireForm.module.scss';
 
+const MAX_LENGTH = 500;
+
 export default function InquireForm(): JSX.Element {
-  const [content, setContent] = useState('');
-  const maxLength = 500;
-  const [isSecret, , , toggle] = useBooleanState(false);
+  const submit = useSubmitInquiry();
+  const location = useLocation();
+
+  const [inquiry, setInquiry] = useState<SubmitInquiry>({
+    title: '',
+    content: '',
+    inquiryImages: [],
+    isSecret: false,
+  });
+
+  const isAttached = useMemo(
+    () => !!inquiry.inquiryImages && inquiry.inquiryImages.length > 0,
+    [inquiry.inquiryImages],
+  );
```

Comment:
메모이제이션할정도의 값은 아닌 것 같아요 필요한 값인가요?
```suggestion
  const isAttached = inquiry.inquiryImages.length > 0
```

Reply:
콜백과 메모가 그저 좋은 건 줄 알았어요😅 사용 할 때마다 비용이라는 말을 듣고 다 삭제 했습니다.
6f0aa7126aee85824da0b8f7e875569d0c646205